datacenters.csv

```diff
@@ -1282,6 +1286,7 @@
 89.47.16.0,89.47.31.255,everhost.ro,http://everhost.ro/
 89.106.0.0,89.106.31.255,Grid Telekom,http://grid.com.tr/
 89.107.16.0,89.107.23.255,Telecity Group,http://www.ukgrid.net/
+89.111.176.0,89.111.179.255,r01,https://r01.ru/
```

Comment:
@alaz just curious how did you find that this range is r01.ru? I don't see it mentioned in `whois`, and `host` lookups on some of the IPs in the range give me hostnames ending in `hc.ru`

Reply:
From `whois`:

```
route:          89.111.176.0/21
descr:          JSC Centrohost route
origin:         AS41126
mnt-by:         RUNIC-MNT
created:        2015-02-25T15:41:27Z
last-modified:  2015-05-13T15:14:32Z
source:         RIPE
```

[ipinfo on AS41126](https://ipinfo.io/AS41126): related domain is `r01.ru`

<details>
<summary>`centrohost.ru` issues redirect to `r01.ru`</summary>

```
$ curl -s http://centrohost.ru
<html>
<meta http-equiv='cache-control: no-cache, must-revalidate'>
<meta http-equiv='pragma' content='no-cache'>
<meta http-equiv="refresh" content="0; URL=https://r01.ru">
```
</details>

Reply:
@alaz great, thanks!